custom_components/ha-glocaltokens/api.py

```diff
@@ -46,3 +54,66 @@ def get_google_devices_json(self):
     def get_android_id(self):
         """Generate random android_id"""
         return self._client._get_android_id()
+
+    def format_timer_information(self, timer):
+        timestamp = convert_from_ms_to_s(timer[FIRE_TIME])
+        timer[DATE_TIME] = datetime.fromtimestamp(timestamp).strftime(SHOW_TIME_ONLY)
+
+        duration = convert_from_ms_to_s(timer[ORIGINAL_DURATION])
+        timer[DURATION] = datetime.utcfromtimestamp(duration).strftime(SHOW_TIME_ONLY)
+        return timer
+
+    def format_alarm_information(self, alarm):
+        timestamp = convert_from_ms_to_s(alarm[FIRE_TIME])
+        alarm[DATE_TIME] = datetime.utcfromtimestamp(timestamp).strftime(SHOW_DATE_TIMEZONE)
+        alarm[LOCAL_TIME] = datetime.fromtimestamp(timestamp).strftime(SHOW_DATE_AND_TIME)
+        return alarm
+
+    def create_url(self, ip, api_endpoint):
+        url = 'https://{ip}:{port}/{endpoint}'.format(
+            ip=ip,
+            port=str(PORT),
+            endpoint=api_endpoint
+        )
+        return url
+
+    def get_alarms_and_timers_from(self, ip, token, endpoint):
+
+        url = self.create_url(ip, endpoint)
+
+        HEADERS[HEADER_CAST_LOCAL_AUTH] = token
+
+        response = requests.get(url, headers=HEADERS, verify=False, timeout=TIMEOUT)
```

Comment:
Can we please add here a little comment saying that `verify=False` is needed to avoid SSL security checks? It would be a good reminder for someone looking at the code